Java/serialization/src/main/java/com/amazon/randomcutforest/serialize/json/v1/V1JsonToV2StateConverter.java

```diff
@@ -46,6 +48,15 @@ public RandomCutForestState convert(String json, Precision precision) throws IOE
         return convert(forest, precision);
     }
 
+    public RandomCutForestState convert(ArrayList<String> jsons, int numberOfTrees, Precision precision)
+            throws IOException {
+        ArrayList<V1SerializedRandomCutForest> forests = new ArrayList<>();
```

Comment:
suggestion. the size of the array is known jsons.size().

Reply:
If you suggest the rewrite, I am happy to change

Reply:
yes, it's a suggestion for using the arraylist constructor with a known size. or there could be unnecessary overallocation/reallocation.

Reply:
the suggested code is
```
ArrayList<V1SerializedRandomCutForest> forests = new ArrayList<>(jsons.size());
```

Reply:
Done